### PR TITLE
css-box-model feature added

### PR DESCRIPTION
**Pull request by Recdata**
**Description:**
This pull request includes CSS Box Model interactive Visualizer. It helps CSS Enthusiast to learn more about core concept in CSS. It will also help them understanding concept of padding, margin and border.

**Changes:**

1. Added new link to the sidebar menu for better accessibility to the users.
2. Added the CSS Box Model feature in css-box.html file with little ui changes and used vanilla javascript to add interactivity in the feature.

_These changes aim to improve the user experience by providing more interactive learning feature to grasp difficult concepts._

### CODE REVIEW

File: scripts/cssbox.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+var controls = document.querySelectorAll('form input, form select');
```
Comment (fahdi) [inline scripts/cssbox.js:1]: This should only work on document load. Please see other scripts.

File: styles/cssbox.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+box-sizing: var(--box-sizing);
+height: var(--height);
+width: var(--width);
+padding: var(--padding);
+border: solid orange;
+border-width: var(--border);
+}
+
+#box {
+/* method for coloring the padding, as seen at: https://stackoverflow.com/questions/14628601/can-i-add-background-color-only-for-padding*/
```
Comment (fahdi) [inline styles/cssbox.css:124]: Please format the CSS.

**Review by fahdi — CHANGES_REQUESTED** [review]
Small changes and then it's good to go.

**Comment by fahdi** [comment]
A few notes on PR description:

1. Please add a screenshot.

**Comment by fahdi** [comment]
@Recdata Excellent work. I have tested it. I'll merge it for now but please note the review. I'll make these changes myself.

**Comment by Recdata** [comment]
Thanks sir, it means a lot. I will review the changes.

**Pull request merged**